content/webapp/types/exhibition-guides.ts

```diff
@@ -15,6 +15,18 @@ type CaptionsOrTranscripts = {
   context?: prismic.RichTextField;
 };
 
+export type GuideHighlightTour = {
+  number: number | undefined;
+  title: string;
+  audio: string | undefined;
+  transcript: prismic.RichTextField | undefined;
+  audioDuration: number | undefined;
+  video: string | undefined;
+  subtitles: prismic.RichTextField | undefined;
+  videoDuration: number | undefined;
+  image: ImageType | undefined;
+};
```

Comment:
Mega NP/non-blocker/probably a different ticket. 
I think I've asked myself this before because we use both across the codebase; the other types use the `number?: number` format over `number: number | undefined`.
I wonder if we only want to use the latter when we explicitly want to say THIS IS UNDEFINED, and I don't think it should be the case that often? (based on [the info found here](https://levelup.gitconnected.com/when-to-use-the-question-mark-or-undefined-type-in-typescript-233e74ab436))

Reply:
it could 100% be a different ticket to tidy across if we prefer it, so don't feel it's needed here, just wanted to start a chat.
I think it could be useful in the case where we want it very clear it's undefined because _it's an exception_, so case-by-case.

Reply:
> perhaps auto-generated ones use the | undefined syntax?

They might tbf! Good thought.